runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -155,6 +155,24 @@ SETVAL(eq_returnAddressDirectMethod,8)
 SETVAL(eq_resolveVirtualJitEIPForHelperCall,12)
 SETVAL(eq_ResolveVirtualData_RIP,0)
 
+ZZ Interface Call Helper Read Only Layout
+SETVAL(eq_InterfaceCallHelperReadOnly_Data,0)
+SETVAL(eq_InterfaceCallHelperClassPICSlot1,0)
+SETVAL(eq_InterfaceCallHelperMethodPICSlot1,8)
+SETVAL(eq_InterfaceCallHelperClassPICSlot2,16)
+SETVAL(eq_InterfaceCallHelperMethodPICSlot2,24)
+SETVAL(eq_InterfaceCallHelperClassPICSlot3,32)
+SETVAL(eq_InterfaceCallHelperMethodPICSlot3,40)
+SETVAL(eq_InterfaceCallHelper_cpAddress,48)
+SETVAL(eq_InterfaceCallHelper_cpIndex,56)
+SETVAL(eq_InterfaceCallHelper_interfaceClass,64)
+SETVAL(eq_InterfaceCallHelper_methodIndex,72)
+SETVAL(eq_InterfaceCallHelperTotalPICSlotSize,88)
+SETVAL(eq_InterfaceCallHelper_isCacheFullFlag,96)
```

Comment:
We should try to keep the names of these the same as the fields for easier searchability. For example searching for "totalSizeOfPicSlots" will not get a hit here.

Reply:
Fixed in https://github.com/eclipse/openj9/compare/ab82c5d254e0315b3c65722371fda75cc1d79a10..3282a01df297676d33999fca192d2613b5db1b09

Reply:
Fixed in https://github.com/eclipse/openj9/compare/ab82c5d254e0315b3c65722371fda75cc1d79a10..3282a01df297676d33999fca192d2613b5db1b09